uvahelpr_web/marketplace/urls.py

```diff
@@ -3,5 +3,6 @@
 
 from . import views
 urlpatterns = [
-    url(r'^$', views.index, name='index')
+    url(r'^$', views.index, name='index'),
+    url(r'^jobs/', views.allJobs, name='allJobs'),
```

Comment:
Add $ at the end  of "^jobs/" so that a url like "...jobs/..." doesn't get captured.